contracts/transfer/src/state.rs

```diff
@@ -379,6 +399,12 @@ impl TransferState {
         }
     }
 
+    /// Return the modules' balances map
+    /// todo: check if this method can be eliminated
+    pub fn balances(&self) -> &BTreeMap<ModuleId, u64> {
+        &self.balances
+    }
```

Comment:
We could make this a formal contract query, but returning the whole map is likely unwise.

Perhaps we can make it a query taking a module ID as an argument and producing a `u64` as a return? Sounds useful for upstream as well.